components/script/dom/subtlecrypto.rs

```diff
@@ -359,6 +500,86 @@ fn normalize_algorithm(
 }
 
 impl SubtleCrypto {
+    /// <https://w3c.github.io/webcrypto/#aes-cbc-operations>
+    fn encrypt_aes_cbc(
+        &self,
+        params: SubtleAesCbcParams,
+        key: &CryptoKey,
+        data: &[u8],
+    ) -> Result<*mut JSObject, Error> {
```

Comment:
This method should take a MutHandleObject instead of returning a raw JSObject pointer.